src/SignalRClient/SignalRClient.csproj

```diff
@@ -2,13 +2,13 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>netcoreapp5.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.SignalR.Protocols.MessagePack" Version="3.1.0" />
-    <PackageReference Include="Microsoft.AspNetCore.SignalR.Client" Version="3.1.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="3.1.0" />
+    <PackageReference Include="Microsoft.AspNetCore.SignalR.Protocols.MessagePack" Version="5.0.0-*" />
```

Comment:
Do we do this in other benchmarks?

Reply:
I think the majority of the other benchmarks have references to things in the shared framework.

Reply:
EF?

Reply:
The EF team updates the version manually once a week ... not sure if it's better than this. Could we report this version in the benchmarks results? We can't even assume it will match the aspnet runtime version as it could be set by the server while this one will look at the feeds.

Reply:
Should I be using the dependencies.props file?

Reply:
Uhhhh, I think this is fine since this is the behavior I want.

Reply:
> Could we report this version in the benchmarks results?

Any suggestions on how to do this?

Reply:
https://github.com/aspnet/Benchmarks/blob/master/src/Benchmarks/Program.cs#L53-L66